### DISCUSSION TITLE
Set first line indent

### DISCUSSION THREAD

**Discussion by Hu-Jianxiong** [davidgohel/officer#416] [Q&A]
I'm using office to generate word reports and I have to say that office does a great job. However, I want to set the indent of the first line of the paragraph, and I have tried for a long time, but I don't know how to set it.

**Comment by davidgohel** [comment]
Sorry for the delay - I did not see your post,

You can't programmatically set the first ident with officer (by using `fp_par()`) but you can use a style from the template that use first indent.
